### PR TITLE
JOI 20-joitter explanation + code without templates

### PR DESCRIPTION
**Pull request by dolphingarlic**
(Also, added BOI 19-tennis to Aliens Trick)

### CODE REVIEW

**Comment by dolphingarlic** [comment]
Also, should I remove the comments from my code? They don't really match up with what I say in the editorial I think

**Review by bqi343 — CHANGES_REQUESTED** [review]
looks fine

File: solutions/JOI 20-joitter.mdx [hunk 1]
```diff
@@ -2,257 +2,157 @@
...
+1. We need to subtract the number of followings from **inside** the previously-unmerged $A$ and $B$. Since each pair of people followed each other, this number is $|A| \cdot (|A| - 1) + |B| \cdot (|B| - 1)$.
+2. We need to add the number of followings from **inside** the newly-merged $A \cup B$. Since each pair of people will follow each other, this number is $|A \cup B| \cdot (|A \cup B| - 1)$.
+3. We need to add the number of followings from **outside** $A \cup B$. Since people who previously followed everyone in $A$ will now also follow everyone in $B$ and vice-versa, this number is $|A| \cdot |F_B| + |B| \cdot |F_A|$.
+4. We need to subtract the number of followings from people who previously already followed everyone in both $A$ and $B$ or were already in $A$ or $B$. This number is $|(A \cup F_A) \cap (B \cup F_B)| \cdot |A \cup B|$.
+
+To make calculations a bit easier, let $C_S = S \cup F_S$.
+
+Putting it all together, we get
+$$
+V_i = V_{i - 1} + |A| \cdot |C_B| + |B| \cdot |C_A| - |C_A \cap C_B| \cdot |A \cup B|
```
[the remark this answers is not in the document]
Comment (dolphingarlic) [reply solutions/JOI 20-joitter.mdx:65]: Should I convert it to inline latex?

[the remark this answers is not in the document]
Comment (bqi343) [reply solutions/JOI 20-joitter.mdx:65]: I think it's good to have it on its own line.

**Pull request merged**